spec/integration/option_parser_spec.rb

```diff
@@ -14,7 +14,7 @@
             options do
               include_options do
                 allowed schema
-                defaults %i[foo bar]
+                default %i[foo bar]
```

Comment:
was there a reason for changing `defaults` to `default`? I guess we could also keep it as it was?

Reply:
Thanks, fixed it. Was part of the DSL change we intended to do.